homeassistant/helpers/service.py

```diff
@@ -630,12 +636,34 @@ async def async_get_all_descriptions(
 
```

Comment:
Just a note that the code can be simplified a bit by turning above `if description is None` to `if description is not None: continue`